db/migrations/20250710170208_add-username-to-users.up.sql

```diff
@@ -0,0 +1,22 @@
+BEGIN;
+
+-- Add username column to users table
+ALTER TABLE users
```

Comment:
Not so sure about this. Are we doing this just to avoid a JOIN with the account_providers table? What if the user has multiple account providers?